statistics/handle/handle.go

```diff
@@ -65,11 +65,19 @@ const (
 
 // Handle can update stats info periodically.
 type Handle struct {
+
+	// initStatsCtx is the ctx only used for initStats
+	initStatsCtx sessionctx.Context
+
 	mu struct {
 		sync.RWMutex
 		ctx sessionctx.Context
 		// rateMap contains the error rate delta from feedback.
 		rateMap errorRateDeltaMap
+	}
```

Comment:
I think that `rateMap` can have its own mutex, too. Maybe we can do that in the PR or in another PR, either is ok for me.

Reply:
we will change it later